### PR TITLE
feat: onboarding the Containerized hello world tab with backend sample  

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

File: hello-world-tab-docker/src/components/App.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                ...teamsLightTheme,
+                colorNeutralBackground3: "#eeeeee",
+              }
+        }
+        style={{ background: tokens.colorNeutralBackground3 }}
+      >
+        <Router>
+          {!loading && (
+            <Routes>
+              <Route path="/privacy" element={<Privacy />} />
```
Comment (huimiu) [inline hello-world-tab-docker/src/components/App.tsx:54]: Perhaps we could remove the routes related to privacy and termsofuse.

File: hello-world-tab-docker/appPackage/manifest.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.16",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com",
```
Comment (huimiu) [inline hello-world-tab-docker/appPackage/manifest.json:10]: Should these three URLs have different values?

File: hello-world-tab-docker/src/components/App.tsx (hunk 1 above)
Comment (hund030) [reply hello-world-tab-docker/src/components/App.tsx]: Good catch. removed.

File: hello-world-tab-docker/src/components/App.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  teamsLightTheme,
+  teamsDarkTheme,
+  teamsHighContrastTheme,
+  tokens,
+} from "@fluentui/react-components";
+import { useEffect } from "react";
+import { HashRouter as Router, Navigate, Route, Routes } from "react-router-dom";
+import { app } from "@microsoft/teams-js";
+import { useTeamsUserCredential } from "@microsoft/teamsfx-react";
+import Privacy from "./Privacy";
```
Comment (huimiu) [inline hello-world-tab-docker/src/components/App.tsx:13]: These two imports also need to be removed.

Comment (hund030) [reply hello-world-tab-docker/src/components/App.tsx]: Removed. Thank you.

**Review by huimiu — APPROVED** [review]

**Pull request merged**